GamePlaySystem/src/Script/ExampleScript.h

```diff
@@ -0,0 +1,19 @@
+#ifndef  __EXAMPLESCRIPT_H__
+#define  __EXAMPLESCRIPT_H__
+#include "Script.h"
+
+class ExampleScript : public Script
+{
+public:
+	ExampleScript();
+	~ExampleScript();
+	void Update() override;
+	void OnInspector(ImGuiContext*) override;
+	void Test();
+
+	int i = 0;
```

Comment:
What is `i` ?

Reply:
Just a variable to test the input loading in the .dll. 
It's already deprecated.